test/autests/gold_tests/http2_frames/http2_frames.test.py

```diff
@@ -0,0 +1,38 @@
+'''
+Specify h2 frame sequence.
+'''
+# @file
+#
+# Copyright 2022, Verizon Media
+# SPDX-License-Identifier: Apache-2.0
+#
+
+Test.Summary = '''
+Specify h2 frame sequence.
+'''
+
+#
+# Test 1: Abort after DATA frame
+#
+r = Test.AddTestRun("Specify h2 frame sequence")
+client = r.AddClientProcess("client1", "http2_frames.yaml")
+server = r.AddServerProcess("server1", "http2_frames.yaml")
+proxy = r.AddProxyProcess("proxy1", listen_port=client.Variables.https_port,
+                          server_port=server.Variables.https_port,
+                          use_ssl=True, use_http2_to_2=True)
+
+client.Streams.stdout += Testers.ContainsExpression(
+    'client-request node frame sequence: HEADERS DATA',
+    "client-request node frame sequence")
+
+client.Streams.stdout += Testers.ContainsExpression(
+    'client-request node frame sequence: HEADERS DATA',
+    "proxy-response node frame sequence")
+
+server.Streams.stdout += Testers.ContainsExpression(
+    'server-response node frame sequence: HEADERS DATA',
+    "server-response node frame sequence")
+
+server.Streams.stdout += Testers.ContainsExpression(
+    'server-response node frame sequence: HEADERS DATA',
+    "proxy-request node frame sequence")
```

Comment:
Stepping back, I think these frames sequence debug messages will be a bit noisy. I know you're using this for verification, but now that you've done it, I think I'd rather just remove them if they seem out of place. We should be able to verify the sequence more significantly via the test proxy.

Can you slack me the debug output from parsing for this test so I can see how it looks?